params/hooks_libevm.go

```diff
@@ -79,15 +82,18 @@ var PrecompiledContractsGranite = map[common.Address]vm.PrecompiledContract{
 	nativeasset.GenesisContractAddr:    makePrecompile(&nativeasset.DeprecatedContract{}),
 	nativeasset.NativeAssetBalanceAddr: makePrecompile(&nativeasset.DeprecatedContract{}),
 	nativeasset.NativeAssetCallAddr:    makePrecompile(&nativeasset.DeprecatedContract{}),
+	P256VerifyAddress:                  vm.PrecompiledContractsP256Verify[P256VerifyAddress],
 }
 
-func init() {
-	for addr, precompile := range vm.PrecompiledContractsP256Verify {
-		PrecompiledContractsGranite[addr] = precompile
-	}
+// ActivePrecompiles implements the RulesHooks interface
+func (r RulesExtra) ActivePrecompiles(existing []common.Address) []common.Address {
+	addresses, _, _ := r.ActivePrecompilesWithContract(existing, nil)
+	return addresses
 }
 
-func (r RulesExtra) ActivePrecompiles(existing []common.Address) []common.Address {
+// ActivePrecompiles specifies precompiles that were activated prior to the
```

Comment:
Purely informational as I think this method should be removed.

This comment is unclear and doesn't seem to mirror what the code actually does. It leaves a number of open questions:

1. What do you mean by "specifies"?
2. What does the `addr` argument do?
3. What are the returned arguments?

While you don't have to always specify everything explicitly (I'm not expecting JSDoc-style comments), in this case it's not clear to a caller what anything does.

Reply:
resolving in favor of other comment.